flow/connectors/postgres/validate.go

```diff
@@ -130,3 +131,20 @@ func (c *PostgresConnector) CheckReplicationConnectivity(ctx context.Context) er
 
 	return conn.Close(ctx)
 }
+
+func (c *PostgresConnector) CheckPublicationCreationPermissions(ctx context.Context, srcTableNames []string) error {
+	// Check if we can create a publication
+	// First create a dummy publication
+	pubName := "_peerdb_unused_publication_" + shared.RandomString(5)
```

Comment:
```suggestion
	pubName := "_peerdb_unused_publication_" + shared.RandomString(5)
```

Open to naming this something else cc @iskakaushik

Reply:
maybe `_peerdb_tmp_test_publication_`